src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -461,7 +461,7 @@ internal bool IsPrimaryProject(ProjectId projectId)
             return true;
         }
 
-        protected override bool CanApplyCompilationOptionChange(CompilationOptions oldOptions, CompilationOptions newOptions, CodeAnalysis.Project project)
+        public override bool CanApplyCompilationOptionChange(CompilationOptions oldOptions, CompilationOptions newOptions, CodeAnalysis.Project project)
```

Comment:
Did you ultimately find anything interesting for why we nevr made this public?

Reply:
nope!  we just call it internally righ tnow.  but there's no reason to keep it that way (esp. since we expose the ParseOptions version).